README.md

```diff
@@ -4,23 +4,22 @@
 
 # Introduction
 
-This is a pytorch implementation of a sequence to sequence learning toolkit for the i-machine-think project. This repository is a fork from the pytorch-seq2seq library developed by IBM, but has substantially diverged from it after heavy development. For the original implementation, visit [https://github.com/IBM/pytorch-seq2seq](https://github.com/IBM/pytorch-seq2seq).
+This is a pytorch implementation of a sequence to sequence learning toolkit for the i-machine-think project. This repository was originally a fork from the pytorch-seq2seq library developed by IBM, but has substantially diverged from it after heavy development. For the original implementation, visit [https://github.com/IBM/pytorch-seq2seq](https://github.com/IBM/pytorch-seq2seq).
 
 # Requirements
 
-This library runs with PyTorch 0.4.0. We refer to the [PyTorch website](http://pytorch.org/) to install the right version for your environment.
+This library runs with PyTorch 1.0.0. We refer to the [PyTorch website](http://pytorch.org/) to install the right version for your environment.
 To install additional requirements (including numpy and torchtext), run:
 
 `pip install -r requirements.txt`
 
 # Quickstart
 
-To use machine, clone the repository and do an editable install with pip.
-Although machine is compatible with python2.7, we recommend you use python3.
+To use machine, clone the repository and do an editable install with pip. Note that `machine` requires python 3.5 or higher (python 2.7 is no longer supported).
 
 ```
 git clone https://github.com/i-machine-think/machine.git $path_to_machine
-pip3 install $path_to_machine
+pip install $path_to_machine
```

Comment:
What is the reason you changed this to pip instead of pip3? It may be a difference between using virtualenv and conda? (if I do pip install on my machine it installs for python2)

Reply:
Yes indeed this would not be good when using pip without a virtual python environment. I will change that command back. 

That being said, I would really recommend using conda and not ever calling pip without a virtual env.

Reply:
I've changed the test to be `resume_training`

Reply:
I think even within a virtual environment you need to specify the python version (at least for me it does)

Reply:
I've got a conda env with python 3, so when it is activated calling pip or python simply defaults to the currently activated environment ie python 3, and I think pipenv works the same. But ok just changed it back because I guess `python3` and `pip3` do work as well in python 3 so it's not actually limiting.

Reply:
Hmm maybe I should revisit my practices, I'm using venv but there it is actually required to specify which version of python to use even after activating the environment. Thanks for changing it back :)

Reply:
Also I agree about coverage, I can probably remove `coverage report -m` because it's even calling it on files in external libraries. I'm running now omitting those files to see if that helps.